charts/rancher-monitoring/103.0.3+up45.31.1/templates/alertmanager/secret.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if and (.Values.alertmanager.enabled) (not .Values.alertmanager.alertmanagerSpec.useExistingSecret) }}
 {{/* This file is applied when the operation is helm install and the target secret does not exist. */}}
 {{- $secretName := (printf "alertmanager-%s" (include "kube-prometheus-stack.alertmanager.crname" .)) }}
-{{- if or (not (lookup "v1" "Secret" (include "kube-prometheus-stack.namespace" .) $secretName)) (eq .Values.alertmanager.secret.recreateIfExists true) }}
```

Comment:
this also seems like it may need a revert from https://github.com/rancher/charts/commit/3a699797bf682b7a79d96f115ded2e265727bb07#diff-478372751533d3778711f02a022f7264593c0e114c85337c49fc1d2a729d1bba